README.md

```diff
@@ -1,38 +1,42 @@
-[![Build Status](https://api.travis-ci.org/bitronix/btm.png?branch=master)](https://travis-ci.org/bitronix/btm)
+# Bitronix Transaction Manager (BTM)
 
-The master version of the current source was built and published on maven central over [here](https://mvnrepository.com/artifact/com.github.marcus-nl.btm).
+Bitronix Transaction Manager (BTM) is a simple but complete implementation of the Java Transaction API (JTA) 1.1. This eXtended Architecture (XA) transaction manager provides all services required by the JTA while keeping the code as simple as possible for easier understanding of XA semantics.
 
-### Help!
-![help wanted](https://static.wixstatic.com/media/7dc740_3a94c9b3ccd14f939b7595843dc7c72d~mv2.png/v1/fill/w_240,h_240,al_c/7dc740_3a94c9b3ccd14f939b7595843dc7c72d~mv2.png)
+The master version of the current source was built and published on the [Maven Central repository](https://mvnrepository.com/artifact/com.github.marcus-nl.btm).
 
-BTM is looking for a new motivated team to look after it. If you would like to take over the project, please by all means contact us: [@bitronix](https://twitter.com/bitronix) and [@BrettWooldridge](https://twitter.com/BrettWooldridge).
+## What's new
```

Comment:
I think we can remove the section because it's not really new anymore.
But, I want to keep in the release tag.
BTW, I noticed there is not 3.0 tag in the releases...
So, later, we can tag 3.0 and copy the section to the release note.

Reply:
I agree. For now, I've removed this section. 

I've also created a [draft of the 3.0.0 release](https://github.com/scalar-labs/btm/releases/tag/untagged-8b17e22130fcaaa0052c) that includes the contents of the `## What's new` section.

Reply:
That's great. Thank you!  🙇